apps/web/src/components/Table/ClientManagedTable/tableCommaList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface TableCommaListProps {
+  list: string[];
+}
+
+const TableCommaList = ({ list }: TableCommaListProps) => {
+  return list.length > 0 ? (
+    <p>
+      {list.map((item, index) => {
```

Comment:
You could use array.join to do this for you.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
Oh ya 👍 thanks! Fixed by [ed7e4de](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7101/commits/ed7e4de78d0094ed66f803b3079dbc9905ef7691)